tests/repro_tests/frontend/detector_descriptor/test_detector_descriptor_reproducibility_base.py

```diff
@@ -0,0 +1,36 @@
+""""Reproducibility tests for detector-descriptors.
+
+Authors: Ayush Baid
+"""
+import unittest
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+
+import gtsfm.utils.io as io_utils
+from gtsfm.common.image import Image
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.detector_descriptor.detector_descriptor_base import DetectorDescriptorBase
+from gtsfm.frontend.detector_descriptor.sift import SIFTDetectorDescriptor
+from tests.repro_tests.test_repro_base import ReproducibilityTestBase
+
+# defining the path for test data
+TEST_DATA_ROOT_PATH = Path(__file__).resolve().parent.parent.parent.parent / "data"
+IMG_PATH = TEST_DATA_ROOT_PATH / "set1_lund_door" / "images" / "DSC_0001.JPG"
+
+RESULT_TYPE = Tuple[Keypoints, np.ndarray]
```

Comment:
maybe `DET_DESC_RESULT_TYPE` is clearer? or maybe not. Seems a bit less generic, but it's more wordy.

Reply:
resolved, thanks